client/my-sites/stats/stats-module/index.jsx

```diff
@@ -152,66 +154,103 @@ class StatsModule extends Component {
 			'is-refreshing': requesting && ! isLoading,
 		} );
 
+		const shouldHideOldModule = isEnabled( 'stats/new-stats-module-component' ) && path === 'posts';
+
 		return (
-			<div>
-				{ siteId && statType && (
-					<QuerySiteStats statType={ statType } siteId={ siteId } query={ query } />
-				) }
-				{ ! isAllTime && (
-					<SectionHeader
-						className={ headerClass }
-						label={ this.getModuleLabel() }
-						href={ ! summary ? summaryLink : null }
+			<>
```

Comment:
The changes here are hard to parse due to git butchering the diff, unfortunately.

Relocating `QuerySiteStats` here may make the diff easier to read:

```suggestion
			<>
				{ siteId && statType && (
					<QuerySiteStats statType={ statType } siteId={ siteId } query={ query } />
				) }
```

Reply:
I moved it manually, thanks for the suggestions. Let me know if this helps.

Reply:
Helps a ton, thanks!